### PR TITLE
WT-7415 Add new configuration files with backup option

### PR DESCRIPTION
**Pull request by jiechenbo**
This ticket involves adding new wtperf configurations as part of testing backup. The idea is to add backup to existing configurations and doing analysis on the original configuration and new configuration with backup. Will need to add the configs to jenkin after PR is merged

### CODE REVIEW

**Comment by jiechenbo** [comment]
evergreen retry

File: bench/wtperf/runners/evict-btree-readonly-backup.wtperf [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+icount=10000000
+report_interval=5
+run_time=120
+populate_threads=1
+readonly=true
+threads=((count=16,reads=1))
+# Add throughput/latency monitoring
+max_latency=2000
+sample_interval=5
+backup_interval=40
```
Comment (quchenhao) [inline bench/wtperf/runners/evict-btree-readonly-backup.wtperf:13]: Add an extra line at end

**Review by quchenhao — APPROVED** [review]
I suppose the configurations are somewhere already and we are just moving them here?

**Comment by jiechenbo** [comment]
I think the idea here is to, create new backup configurations and once we are happy with configs, we merge then place them into jenkins @hbokhari  is that the idea?

**Comment by kommiharibabu** [comment]
@quchenhao These are the new wtperf configuration files with backup configuration similar to the existing ones.

File: bench/wtperf/runners/evict-btree-readonly-backup.wtperf (hunk 1 above)
Comment (jiechenbo) [reply bench/wtperf/runners/evict-btree-readonly-backup.wtperf]: ta!

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Pull request merged**